AlgorithmiaTests/AlgorithmiaTests.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  AlgorithmiaTests.swift
+//  AlgorithmiaTests
+//
+//  Created by Erik Ilyin on 11/9/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import XCTest
+@testable import Algorithmia
+
+class AlgorithmiaTests: XCTestCase {
+    var client:AlgorithmiaClient?
+    override func setUp() {
+        super.setUp()
+        
+        let ALGORITHMIA_API_KEY = "" // Your API Key
```

Comment:
Switched to env var in 243d5b51

That said, it's worth noting I discussed this a bit with Erik, and looked into a bit myself, and the degree to which swift is tightly coupled with XCode makes this more awkward than our other clients. A good example that inspired Erik's final piece is what Algolia did with [their swift client's apiKey](https://github.com/algolia/algoliasearch-client-swift/search?utf8=%E2%9C%93&q=PLACES_API_KEY&type=Code)